framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2631,6 +2631,17 @@ public final boolean shouldSkipDefs(ClassTree tree) {
         || !onlyDefsPattern.matcher(qualifiedName).find();
   }
 
+  /**
+   * Tests whether the method definition should not be checked because it matches the {@code
+   * checker.skipDefs} property.
+   *
+   * @param tree method to potentially skip
+   * @return true if checker should not test {@code tree}
+   */
+  public boolean shouldSkipDefs(MethodTree tree) {
```

Comment:
Instead of passing in a command-line flag, classes are able to override this method to specify which method should (or should not) be checked.